test/web-platform-tests/to-run.yaml

```diff
@@ -1,5 +1,7 @@
 DIR: FileAPI
 
+Blob-methods-from-detached-frame.html: [fail-slow, iframe onload timing]
+BlobURL/cross-partition.tentative.https.html: [fail-slow, iframe onload timing]
```

Comment:
The intent of fail-slow is that, if we ran this test, it would fail with a non-timeout error, but it would take so long to fail that we shouldn't bother.

Whereas the intent of timeout is, if we ran this test, it would fail with a timeout error.

It's possible we haven't applied these consistently so far, but you could imagine a CI run which runs all the skipped tests and checks that we're using the right status...

(Although, I'm not sure if there's a lot of value in separating these anymore, hmm.)

Anyway, don't stress about getting it exactly right, but if there's clear cases where timeout is more correct, switching would probably be good. Or maybe we should just work on a separate PR to get rid of "timeout" and replace it with "fail-slow" everywhere, after this merges...